### PR TITLE
[DH-655] Event filters 

### PR DESCRIPTION
**Pull request by elcct**
This adds following filters to event: name, organiser name, address country, start date.

### CODE REVIEW

File: datahub/search/dsl_utils.py [hunk 1]
```diff
@@ -22,6 +22,18 @@ def contact_or_adviser_mapping(field, include_dit_team=False):
     return Nested(properties=props)
 
 
+def contact_or_adviser_partial_mapping(field):
+    """Mapping for Adviser/Contact fields that allows partial matching."""
+    props = {
+        'id': KeywordString(),
```
Comment (marcofucci) [inline datahub/search/dsl_utils.py]: should this be `index='no'` instead of `not_analyzed`?
We are probably never gonna search for a related id, no?

Comment (marcofucci) [reply datahub/search/dsl_utils.py]: Ignore me, we might want to filter based on it

Comment (elcct) [reply datahub/search/dsl_utils.py]: We do search other items by `id`, I could see the case when we would want to find event by exact organiser - for example if we get to have auto complete dropdown.

Comment (elcct) [reply datahub/search/dsl_utils.py]: I didn't see your second comment when was replying to your first one :)

**Pull request merged**